src/infra/dex/balancer/dto.rs

```diff
@@ -1,58 +1,208 @@
 use {
     crate::{
-        domain::{auction, dex, order},
+        domain::{dex, eth, order},
+        infra::dex::balancer::Error,
         util::serialize,
     },
+    bigdecimal::BigDecimal,
     ethereum_types::{H160, H256, U256},
+    number::conversions::{big_decimal_to_u256, u256_to_big_decimal},
     serde::{Deserialize, Serialize},
     serde_with::serde_as,
 };
 
-#[derive(Debug, Serialize)]
+pub mod get_swap_paths_query {
```

Comment:
Why do we have this module if there is only a single public item in there?

Reply:
No need anymore. Removed.